buildSrc/src/main/kotlin/io/getstream/chat/android/command/dag/unittest/task/SelectedUnitTestsTask.kt

```diff
@@ -0,0 +1,35 @@
+package io.getstream.chat.android.command.dag.unittest.task
+
+import io.getstream.chat.android.command.dag.parseModules
+import io.getstream.chat.android.command.dag.unittest.filter.unitTestCommand
+import io.getstream.chat.android.command.dag.unittest.plugin.UnitTestsCommandExtesion
+import io.getstream.chat.android.command.utils.changeModuleFileDoesNotExistInPath
+import io.getstream.chat.android.command.utils.writeFile
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+open class SelectedUnitTestsTask : DefaultTask() {
+
+    @Input
+    lateinit var config: UnitTestsCommandExtesion
+
+    @TaskAction
+    private fun command() {
+        val changedModulesFile = File(config.changeModulesPath)
+
+        if (!changedModulesFile.exists()) {
+            changeModuleFileDoesNotExistInPath(config.changeModulesPath)
+        }
+
+        val command = parseModules(changedModulesFile).unitTestCommand(project)
+
+        writeFile(config.outputPath) { writer ->
+            writer.write(command)
+        }
```

Comment:
```suggestion
        File(config.outputPath).writeText(command)
```

Just some simplification again